recirq/third_party/quaff/json_resolver.py

```diff
@@ -4,9 +4,9 @@
 
 
 def json_resolver(quaff_type: str) -> Optional[ObjectFactory]:
-    if quaff_type.startswith("quaff."):
+    if quaff_type.startswith("quaff"):
         return cast(
             ObjectFactory,
-            eval(f"__import__('quaff').{quaff_type.split('.', 1)[1]}", {}, {}),
+            eval(f"__import__('recirq').third_party.{quaff_type}", {}, {}),
```

Comment:
Please rewrite without eval, for example, after adding module-level import `from recirq.third_party import quaff` -
```suggestion
            getattr(quaff, quaff_type[6:]
```

Reply:
Thanks, done.